test/ETHMixer.test.js

```diff
@@ -321,14 +321,14 @@ contract('ETHMixer', accounts => {
       await mixer.deposit(toFixedHex(deposit.commitment), { value, from: sender })
 
       const { root, path_elements, path_index } = await tree.path(0)
-      const oneEtherFee = bigInt(1e18) // 1 ether
+      const largeFee = value + 1
```

Comment:
why do you want to concatenate strings here?
`1000000000000000000` + 1 = `10000000000000000001`

Reply:
fixed